bin/docker-entrypoint

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+dockerize -wait tcp://db:5432 -timeout 20s -wait-retry-interval 10s bash -c "bin/rails db:create"
```

Comment:
I believe that db:create and parallel:create (and maybe even db:seed) might be unnecessary when using db:prepare and parallel:prepare, but I'm not sure. Fine with this as is, but if you wanted to look into it! If that were true that could let us remove some commands from bin/setup as well.